src/library_browser.js

```diff
@@ -19,21 +19,20 @@ var LibraryBrowser = {
 #endif
 #endif
   ],
-  $Browser__postset: `
-    // exports
-    Module["requestFullscreen"] = Browser.requestFullscreen;
+
+  $requestFullscreen: 'Browser.requestFullscreen',
 #if ASSERTIONS
-    Module["requestFullScreen"] = Browser.requestFullScreen;
+  // Legacy name for requestFullscreen (with uppercase S)
+  $requestFullScreen: 'Browser.requestFullScreen',
 #endif
-    Module["requestAnimationFrame"] = Browser.requestAnimationFrame;
-    Module["setCanvasSize"] = Browser.setCanvasSize;
-    Module["pauseMainLoop"] = Browser.mainLoop.pause;
-    Module["resumeMainLoop"] = Browser.mainLoop.resume;
-    Module["getUserMedia"] = Browser.getUserMedia;
-    Module["createContext"] = Browser.createContext;
-    var preloadedImages = {};
-    var preloadedAudios = {};`,
-
+  $requestAnimationFrame: 'Browser.requestAnimationFrame',
+  $setCanvasSize: 'Browser.setCanvasSize',
+  $pauseMainLoop: 'Browser.mainLoop.pause',
+  $resumeMainLoop: 'Browser.mainLoop.resume',
+  $getUserMedia: 'Browser.getUserMedia',
+  $createContext: 'Browser.createContext',
```

Comment:
Do we automatically add helpful messages in debug builds for these? I am guessing maybe no, since I'm not sure we have a list that includes these particular functions atm. We have that for other things where the error message tells the user exactly what to do, which could be good here as well.

Reply:
Yes, we do.  In debug builds if you try to call `Module['anyLibraryFuncAtAll']` you will see this one of these two errors:

https://github.com/emscripten-core/emscripten/blob/04a0cad4d4c9f5d62876821274d78cd0a52427af/src/runtime_debug.js#L65

https://github.com/emscripten-core/emscripten/blob/04a0cad4d4c9f5d62876821274d78cd0a52427af/src/runtime_debug.js#L92

I locally verified this was the case with these changes, but we do have tests for that too.